.github/workflows/e2e.yml

```diff
@@ -50,4 +50,5 @@ jobs:
 
     uses: fluencelabs/flox/.github/workflows/tests.yml@main
     with:
-      js-client-snapshots: "${{ needs.js-client.outputs.js-client-snapshots }}"
\ No newline at end of file
+      js-client-snapshots: "${{ needs.js-client.outputs.js-client-snapshots }}"
+      nox-image: "fluencelabs/nox:unstable_minimal"
```

Comment:
while you are there - please add a new line at the end of the file. We mostly kept new lines in the end for consitency sake and just to never see github complaining about the missing new lines